app/src/main/java/io/intrepid/contest/screens/contestcreation/NewContestActivity.java

```diff
@@ -129,4 +154,13 @@ public void setCategories(List<Category> categories) {
     public void setContestDescription(String description) {
         presenter.setContestDescription(description);
     }
+
+    @Override
+    public void setNextEnabled(boolean enabled) {
+        if (enabled) {
+            presenter.onNextEnabled();
```

Comment:
you can just call this presenter.setNextEnabled and pass the boolean to the presenter.